templates/zerver/integrations.html

```diff
@@ -533,6 +538,33 @@
       <p><small>Thanks to Wes of TurboVote for <a href="https://gist.github.com/cap10morgan/5100822">submitting this integration</a>!</small></p>
     </div>
 
+    <div id="circleci" class="integration-instructions">
+
+      <p>Zulip supports integration with CircleCi and can notify you of
```

Comment:
Should spell it CircleCI since that's how they spell it here and in the next few lines